src/brasil/gov/agenda/browser/agendadiaria.py

```diff
@@ -37,13 +43,18 @@ def update(self):
     def _format_time(self, value):
         return value.strftime('%Hh%M')
 
-    def _translate(self, msgid):
+    def _translate(self, msgid, locale='plonelocales', mapping=None):
         tool = self._ts
+        # XXX: Por que é retornado 'pt-br' do portal_state ao invés de 'pt_BR'?
```

Comment:
Please check [this comment](https://github.com/plonegovbr/brasil.gov.portlets/pull/1#issuecomment-93834355) about comments in English. Thanks!